packages/@okta/i18n/src/properties/login_ok_PL.properties

```diff
@@ -413,11 +413,11 @@ enroll.webauthn.instructions.noSupportForBiometric = 》Ñöţé∶ Šöɱé ƀ
 authfactor.webauthn.error = 》Ýöû åŕé çûŕŕéñţļý ûñåƀļé ţö ûšé å Šéçûŕîţý ķéý öŕ ƀîöɱéţŕîç åûţĥéñţîçåţöŕ· Ţŕý åĝåîñ· ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 enroll.totp.enterCode = 》Éñţéŕ çöðé ðîšþļåýéð ƒŕöɱ ţĥé åþþļîçåţîöñ €홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 enroll.totp.setupApp = 》Ļåûñçĥ โ {0} öñ ýöûŕ ɱöƀîļé ðéṽîçé åñð šéļéçţ ″Åðð åñ åççöûñţ″· Šçåñ ţĥé ǪŔ çöðé ţö çöñţîñûé· €홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
-enroll.totp.setupGoogleAuthApp = 》Ļåûñçĥ ฐโ {0}، ţåþ ţĥé ″⁺″ îçöñ، ţĥéñ šéļéçţ ″Šçåñ ƀåŕçöðé″· 한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
+enroll.totp.setupGoogleAuthApp = 》Ļåûñçĥ ฐโ {0}، ţåþ ţĥé ″⁺″ îçöñ، ţĥéñ šéļéçţ ″Šçåñ å ǪŔ çöðé″· 홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
```

Comment:
Looks like i have missed out in running pseudo-lock couple of months back. Original changes [here](https://github.com/okta/okta-signin-widget/commit/2f09584f022820474261a8c78957139fd5ec3b71#diff-391ef791333e16bfeed711da3def133ca2471a6ebbe1e917ecc61330a1530fe3)